### PR TITLE
Restructure docs directory

### PR DESCRIPTION
**Pull request by philipnbbc**
# Details
Move all templates to a docs/templates directory. Remove decisions/ path component from ADRs

# Pivotal Story (if relevant)
Story URL: https://www.pivotaltracker.com/story/show/187412553

# Related PRs
_Where appropriate. Indicate order to be merged._

# Submitter PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] API version has been incremented if necessary
- [ ] ADR status has been updated, and ADR implementation has been recorded
- [ ] Documentation updated (README, etc.)
- [ ] PR added to Pivotal story (if relevant)
- [ ] Follow-up stories added to Pivotal

# Reviewer PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Design makes sense, and fits with our current code base
- [ ] Code is easy to follow
- [ ] PR size is sensible
- [ ] Commit history is sensible and tidy

# Info on PRs
The checks above are guidelines. They don't all have to be ticked, but they should all have been considered.

### CODE REVIEW

File: docs/adr/0004-ancestry-relationships.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ The discussion concluded that in the general case, this type of ancestry should
 
 Management of complex Source<->Source relationships isn't made possible in an effective way by any of the proposals in this ADR, which only really improves the "lightweight copies" use case.
 For the simple case where you want to make a subclip without modification, it probably makes more sense to store a Source (or Flow) ID and a timerange, e.g. inside a MAM.
-Further to [Deletion of Content and IDs](https://github.com/bbc/tams/blob/main/docs/adr/decisions/0004-content-deletion.md) it is assumed such a MAM would also arbitrate deletion requests to avoid removing content needed by a subclip.
+Further to [Deletion of Content and IDs](https://github.com/bbc/tams/blob/main/docs/adr/0004-content-deletion.md) it is assumed such a MAM would also arbitrate deletion requests to avoid removing content needed by a subclip.
```
Comment (j616) [inline docs/adr/0004-ancestry-relationships.md:91]: Chatting to @samdbmg . Would you mind changing one of the `0004-...` docs to be `0004a-...`? We'll look at putting together a workflow to verify there are no conflicting ADR numbers in future.

**Review by j616 — APPROVED** [review]
One request inline. Otherwise LGTM

File: docs/adr/0004-ancestry-relationships.md (hunk 1 above)
Comment (philipnbbc) [reply docs/adr/0004-ancestry-relationships.md]: Done in https://github.com/bbc/tams/pull/43/commits/72e7bb7baf45546c3c85e69af43cfbc411bb2cba

**Comment by philipnbbc** [comment]
@j616 I've also changed some absolute references to relative ones in https://github.com/bbc/tams/pull/43/commits/cc1ff2fb53638c6557f3a164f0e2d09bc8aa4cd9. Hopefully that's OK.

**Comment by philipnbbc** [comment]
> @j616 I've also changed some absolute references to relative ones in [cc1ff2f](https://github.com/bbc/tams/commit/cc1ff2fb53638c6557f3a164f0e2d09bc8aa4cd9). Hopefully that's OK.

@samdbmg did you intend these references to be targetting specific commits?

**Comment by samdbmg** [comment]
> @samdbmg did you intend these references to be targeting specific commits? I guess that makes sense as things may change...

No, in this case I think that's haphazard copying of links - I'm happy with the change

**Pull request merged**